tests/pytest/run_files/job_jenkins_sarus

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -l
 #SBATCH --job-name="fv3gfs-fortran-test"
 #SBATCH --partition=cscsci
-#SBATCH --account=s1053m
-#SBATCH --constraint=mc
+#SBATCH --account=s1053
+#SBATCH --constraint=gpu
```

Comment:
Why the switch back to GPU?

Reply:
Leftover from me testing both partitions of Daint with the Jenkins plans. I'll switch back to the CPU partition as that was discussed at stand-up.